spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -81,12 +88,18 @@ definitions:
             IOD of the SSR atmospheric correction
 
  - GriddedCorrectionHeader:
-    short_desc: Header for MSG_SSR_GRIDDED_CORRECTION
+    short_desc: Header for the MSG_SSR_GRIDDED_CORRECTION message.
     desc: |
-      The 3GPP message contains nested variable length arrays
+      The LPP message contains nested variable length arrays
       which are not suppported in SBP, so each grid point will
       be identified by the index.
     fields:
+    - tile_set_id:
+        type: u8
+        desc: Unique identifier of the tile set this tile belongs to.
+    - tile_id:
+        type: u8
+        desc: Unique identifier of this tile in the tile set.
```

Comment:
Should this be `u16` to be closer to the LPP spec?

Reply:
@jbangelo I was thinking about this last week. in LPP there is a single 16 bit value. However in SIP we have a 8 bit tile ID, and 8 bit tile group ID.

I was thinking we would combine the two to create a 16 bit ID for LPP?

Reply:
@ljbade Is this resolved with `tile_set_id` and `tile_id`?

Reply:
In SIP the IDs are 32 bits wide. I really think we should make these at least 16 bits. The tile set ID won't be changing much, just when/if we make a new tile layout.